### ISSUE TITLE
binaryen-c.h: this function declaration is not a prototype

### ISSUE THREAD

**Issue by atrosinenko** [WebAssembly/binaryen#1766]
When compiling with Binaryen on Emscripten's clang (related to #1494), I get the following warnings:

```
/home/trosinenko/projects/qemujs/build/emscripten_-m32_-Os_-g2/binaryen/src/binaryen-c.h:655:29: warning: this function declaration is not a prototype [-Wstrict-prototypes]
int BinaryenGetOptimizeLevel();
                            ^
                             void
/home/trosinenko/projects/qemujs/build/emscripten_-m32_-Os_-g2/binaryen/src/binaryen-c.h:663:27: warning: this function declaration is not a prototype [-Wstrict-prototypes]
int BinaryenGetShrinkLevel();
                          ^
                           void
/home/trosinenko/projects/qemujs/build/emscripten_-m32_-Os_-g2/binaryen/src/binaryen-c.h:671:25: warning: this function declaration is not a prototype [-Wstrict-prototypes]
int BinaryenGetDebugInfo();
                        ^
                         void
```

Not too severe, but it disturbs from more significant warnings. Sure I would fix it myself, but analyzing CLA is probably order of magnitude harder than actually fixing :)

https://github.com/WebAssembly/binaryen/blob/da3bb2ffcc8a6efd6a95a07f7de793d1362f4a07/src/binaryen-c.h#L655

**Comment by kripken** [comment]
Thanks @atrosinenko, fix in #1768

**Comment by atrosinenko** [comment]
Thanks!
